Dockerfile

```diff
@@ -1,9 +1,23 @@
-FROM sysprog21/rv32emu-gcc as base_gcc
 FROM sysprog21/rv32emu-sail as base_sail
 
+FROM ubuntu:22.04 as base_gcc
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    build-essential curl && \
```

Comment:
Due to the presence of a git submodule in the source code, it's necessary to install git here in order to successfully execute the subsequent 'RUN make' command.